src/index.js

```diff
@@ -19,6 +26,11 @@ module.exports = function({
   verbose && setVerboseMode();
   switchOff && setSwitchOff();
 
+  if (!isValidToken(token)) {
+    logger.warn('Invalid Token. Go to Lumigo Settings to get a valid token.');
+    setSwitchOff();
+  }
+
```

Comment:
Missing tests to this case, Test the log message with `ConsoleWritesForTesting` and test that the HTTP is not instrumented